spec/runtime/spirits/table-gui@tradeshift.com/ts.ui.TableSpirit.spec.js

```diff
@@ -339,6 +339,17 @@ describe('ts.ui.TableSpirit', function likethis() {
 			});
 		});
 
+		it('should support link in column config', function(done) {
```

Comment:
can we add some tests to make sure that only links are parsed, nothing else?

Reply:
What do you mean? It should parse other whitelisted markdown syntax as well. 🤔 
Before this, if you needed one or two columns of the table have links, you had only the option to make the whole table, `linkable`, which meant beside all those whitelisted, we would also include link syntax for the markdown parser. Which was letting user to provide malicious links in other columns, since developers wouldn't escape all columns. 
Now, with this feature, they are enabling links for one single column, and only do the escaping on that column. The markdown parser is still formatting the table. If they don't need markdown formatting, that should be disabled with `markdownFormatting`.

Reply:
wait, so maybe I misunderstood something...

if I set `markdownFormatting` i can enable/disable markdown... how is that different from `linkable`? :thinking:

Reply:
markdownFormatting need to be enabled, to make the column linkable. If you disable it you won't have a linkable table or col.
With `markdownFormatting`, if you needed a column to support links, you had to disable `markdownFormatting` for all other columns. But in this way, you can only make one column linkable and not worry about other columns being vulnerable to content spoofing.

Reply:
A more convenient way to achieve similar functionality, I would say.

Reply:
Also, `markdownFormatting` and `linkable` solves different problems:
- `markdownFormatting` gives an option to disable markdown processing in particular columns, e.g. if column's content is based on user input and can contain special characters that are harmless but used in markdown (most common are asterisks and backticks).
- `linkable` gives more control on link processing enablement and make it possible to enable it only in columns that are controlled by developers. Previously developer can enable link processing in whole table, now it can be enabled on per-column basis.

But, as @esmailpour-hosein  described, `linkable` is depends on `markdownFormatting` which probably makes a confusion what is a difference.

Reply:
ok, I think i'm getting closer :) so what happens for `markdownFormatting: true, linkable: false`?

Reply:
Those are default values, so this would happen:
<img width="1086" alt="Screenshot 2020-10-26 at 15 40 20" src="https://user-images.githubusercontent.com/22932169/97186530-adeb8900-17a1-11eb-916d-4802b4e633e8.png">

Reply:
so markownFormatting = true means all markdown except links. I think I finally got it :tada: